src/test/java/com/wikia/webdriver/testcases/chattests/ChatTestsStaff.java

```diff
@@ -99,9 +99,31 @@ public void ChatTestsStaff_003_sendPrivateMessage() {
 
   @DontRun(env = {"prod"})
   @Test(groups = {"ChatTestsStaff_004", "ChatStaff", "ChatTests"})
-  public void ChatTestsStaff_004_basicUserChatFails() {
+  public void regularUserCanNotEnterChatOnPreviewEnvironment() {
     openChatForUser(credentials.userName10, credentials.password10);
     SpecialPageObject special = new SpecialPageObject();
     special.verifyPageHeader("Permissions error");
   }
+
+  @DontRun(env = {"sandbox"})
```

Comment:
Courtney chat test can not be run on preview for normal users it can be run only for stuff user this has been set because user were chatting on preview with out mod knowledge 
Chat do not work on sandboxes at all and this is what we are working on now and once is done I will change this in chat tests to